### DISCUSSION TITLE
How can I use colors like purple, orange, teal, etc. in --color flags?

### DISCUSSION THREAD

**Discussion by Caboose12000** [fastfetch-cli/fastfetch#969] [Q&A] [answered]
The only colors I've found that work in the --color flags are black white blue green yellow and red.  Is there a way to spcify color via hex code, or at least a way to see available colors?  I haven't been able to find anything in the docs, so if anyone has any knowledge on this I'd really appreciate it!

**Comment by CarterLi** [comment]
`fastfetch --help color`

**Reply by Caboose12000** [reply]
I think I marked this "answered" too quickly out of embarrassment, sorry 😅 

after running fastfetch --help color, I see one of the examples says `--color 4;92:  sets the color to bright Green with underline` but when I try to run fastfetch with that flag, I get `bash: 92: command not found...` I'm sure I'm still missing something silly here, but I can't figure out what I'm doing wrong

**Reply by CarterLi** [reply, marked as answer]
`fastfetch --color '4;92'`
